src/test/java/org/springframework/samples/petclinic/rest/VisitRestControllerTests.java

```diff
@@ -127,7 +130,10 @@ public void testGetVisitSuccess() throws Exception {
             .andExpect(status().isOk())
```

Comment:
**Hight** Vet field is not checked. Need to add vet field to visits items and check for jsonPath vet as well.